docs-js/docs-nav.js

```diff
@@ -0,0 +1,11 @@
+btnToggle = document.querySelector("#btn-toggle");
+navToggle = document.querySelector("#nav-toggle");
```

Comment:
Avoid this type of variable declaration, as in a large codebase it will create issues. Always declare variable types whenever you are declaring them.